Clarify color weight

svefredrik opened the pull request:
The sum of all ColorCluster weights in a ColorDescriptor shall not exceed 100.

Fixes #122.

HansBusch commented:
Two remarks:
 1. ONVIF is using everywhere normalized ranges from zero to one. Propose to declare the example as wrong.

 2. It seems to me that color quantization is up to the implementer. By limiting the sum to 100 overlaps are not allowed. However things get blurry when the covariance is present. Propose to just define the range.

svefredrik commented:
> Two remarks:
> 
> 1. ONVIF is using everywhere normalized ranges from zero to one. Propose to declare the example as wrong.
> 2. It seems to me that color quantization is up to the implementer. By limiting the sum to 100 overlaps are not allowed. However things get blurry when the covariance is present. Propose to just define the range.

OK, I changed range to [0..1] and updated the examples.

HansBusch commented:
Updated the title of the PR
